README.md

```diff
@@ -6,75 +6,168 @@
 
-# Create resources
-% ks apply -f examples/guestbook.jsonnet
+* *You should have `kubectl` installed.* If not, follow the instructions for [installing via Homebrew (MacOS)](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-homebrew-on-macos) or [building the binary (Linux)](https://kubernetes.io/docs/tasks/tools/install-kubectl/#tabset-1).
 
-# Modify configuration (downgrade gb-frontend image)
-% sed -i.bak '\,gcr.io/google-samples/gb-frontend,s/:v4/:v3/' examples/guestbook.jsonnet
-# See differences vs server
-% ks diff -f examples/guestbook.jsonnet
+* *Your `$KUBECONFIG` should specify a valid `kubeconfig` file*, which points at the cluster you want to use for this demonstration.
 
-# Update to new config
-% ks apply -f examples/guestbook.jsonnet
+### Script
+
+Copy and paste the script below to deploy the container image for a basic web app UI:
+
+```
+# Start by creating your app directory
+ksonnet init quickstart
+cd quickstart
+
+# Autogenerate a basic manifest
+ksonnet generate deployment-exposed-with-service guestbook-ui \
+--name guestbook \
+--image alpinejay/dns-single-redis-guestbook:0.3 \
+--type ClusterIP
+
+# Deploy your manifest to your cluster
+ksonnet apply default
+
+# Set up an API proxy so that you can access the guestbook service locally
+kc proxy > /dev/null &
+PROXY_PID=$!
+QUICKSTART_NAMESPACE=$(kubectl get svc guestbook -o jsonpath="{.metadata.namespace}")
+GUESTBOOK_SERVICE_URL=http://localhost:8001/api/v1/proxy/namespaces/$QUICKSTART_NAMESPACE/services/guestbook
+
+# Check out the guestbook app in your browser (NOTE: the buttons don't work!)
+open $GUESTBOOK_SERVICE_URL
+
+```
+
+The rest of this script upgrades the container image to a new version:
 
-# Clean up after demo
-% ks delete -f examples/guestbook.jsonnet
 ```
+# Bump the container image to a different version
+ksonnet param set guestbook-ui image alpinejay/dns-single-redis-guestbook:0.4
+
+# See the differences between your local manifest and what's running on your cluster
+# (ERROR is expected here since there are differences)
+ksonnet diff local:default remote:default
+
+# Update your cluster with your latest changes
+ksonnet apply default
+
+# (Wait a bit) and open another tab to see newly added javascript
+open $GUESTBOOK_SERVICE_URL
+
+```
+
+Notice that the webpage looks different! Now clean up:
+
+```
+# Teardown
+kill -9 $PROXY_PID
+ksonnet delete default
+
+# There should be no guestbook service left running
+kubectl get svc guestbook
+```
+
+Even though you've made modifications to the Guestbook app and removed it from your cluster, ksonnet still tracks all your manifests locally:
+
+```
+# View all expanded manifests (YAML)
+ks show default
+```
+
+If you're wondering how ksonnet differs from existing tools, the [full-length tutorial (WIP)]() shows you how to use other ksonnet features to implement the rest of the Guestbook app (so that the buttons work!).
```

Comment:
The full-length tutorial goes to a missing page. Is this intentional or is there supposed to be a stub (i.e. to a TODO page similar to concepts)?

Reply:
yes, the tutorial will be hosted on the new ksonnet site, and is not yet completed. if it is easier (and less confusing), I can remove the link for now